osf/management/commands/migrate_registration_responses.py

```diff
@@ -0,0 +1,124 @@
+import datetime
+import logging
+
+from django.core.management.base import BaseCommand
+from django.apps import apps
+from tqdm import tqdm
+
+from bulk_update.helper import bulk_update
+from framework.celery_tasks import app as celery_app
+
+logger = logging.getLogger(__name__)
+
+def migrate_registrations(dry_run, rows='all'):
```

Comment:
for bad/unexpected data, skip and set `migrated` to false, so we can find them later (don't just stop on it)